lib/rules/length-zero-no-unit/__tests__/index.mjs

```diff
@@ -806,38 +812,17 @@ testRule({
 			code: '@detached-ruleset: { font: normal normal 400 0/0px cursive; };',
 			description: 'ignore Less detached ruleset with font',
 		},
-	],
-
-	reject: [
 		{
 			code: '@detached-ruleset: { font-size: 0px; };',
-			fixed: '@detached-ruleset: { font-size: 0; };',
-
-			message: messages.rejected,
-			line: 1,
-			column: 34,
-			endLine: 1,
-			endColumn: 36,
+			description: 'ignore Less detached ruleset with font-size',
```

Comment:
This change will also prevent variables in `@detached-ruleset` from being updated. 
Do you think this is appropriate?

Reply:
Yes, I believe it's fine. These days, we typically prefer to ignore non-standard constructs in the built-in rules.